packages/ag-charts-website/src/content/docs/benchmarks/_examples/enterprise-1M-line-series/data.ts

```diff
@@ -35,7 +35,7 @@ export function getData() {
         currentPrice += (random() * 2 - 1) * maxDailyPriceChange;
         const volume = maxDailyVolume * random();
 
-        const timestamp = new Date(2024, 0, 1, -i);
+        const timestamp = new Date(2024, 0, 1, -i).getTime();
```

Comment:
Was this needed?

Reply:
The options extraction serialises to JSON, which converts `Date` to `string` formats.